app/views/foods/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="min-h-screen bg-gray-100 py-6">
+  <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+    <div class="text-left">
+      <h1 class="font-bold text-center text-4xl bg-blue-500 text-white py-2 rounded-md">Add Food</h1>
+      <%= form_with model: @food, url: user_foods_path, method: :post, data: { turbo: false } do |f| %>
+        <div class="mb-4">
+          <%= f.label :name, "Food name",class: "block text-gray-700 font-bold mb-2" %>
+          <%= f.text_field :name, class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" %>
+        </div>
+        <div class="mb-4">
+          <%= f.label :measurement_unit, class: "block text-gray-700 font-bold mb-2" %>
+          <%= f.text_field :measurement_unit, class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" %>
+        </div>
+        <div class="mb-4">
+          <%= f.label :price, "Unit price",class: "block text-gray-700 font-bold mb-2" %>
+          <%= f.text_field :price, class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" %>
+        </div>
+        <div class="mb-4">
+          <%= f.label :quantity, class: "block text-gray-700 font-bold mb-2" %>
+          <%= f.text_field :quantity, class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" %>
+        </div>
```

Comment:
Please make a number field instead of a text field because if the user didn't submit is not a number. I will be 0.00 which is not good you can change the number field for number input with at least HTML validation.(You can use other validation library and make controller level or model level validation is fine too but it requires some work)